commit 4f91a31f05adb9b5a3c4d044c96837f853e13497
Author: Torsten Dreyer <Torsten@t3r.de>
Date:   Sun Aug 17 22:36:19 2014 +0200

    Quick and dirty repair of linux build with RTI enabled

diff --git a/utils/fgai/AIBVHPager.cxx b/utils/fgai/AIBVHPager.cxx
index a5cc139..9aee79d 100644
--- a/utils/fgai/AIBVHPager.cxx
+++ b/utils/fgai/AIBVHPager.cxx
@@ -18,6 +18,7 @@
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
+#include <osg/Referenced>
 
 #include "AIBVHPager.hxx"
 
